vars/centos-8.yml

```diff
@@ -14,5 +14,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+_rpm_repository:
```

Comment:
`_rpm_repository` is used for initializing yum repo with upstream provided `repo.rpm`, not for installing the dependency nor package itself :-(

Read https://www.darktable.org/install/#rhel and https://www.darktable.org/install/#3rdparty

Reply:
Required pacakge:
- https://centos.pkgs.org/8/centos-powertools-x86_64/lensfun-0.3.2-14.el8.x86_64.rpm.html

Enable PowerTools in our style:
- https://github.com/alvistack/ansible-role-bootstrap/blob/master/vars/centos-8.yml#L42